pkg/util/utils.go

```diff
@@ -305,6 +304,21 @@ func GetImageConfig(changes []string) (ImageConfig, error) {
 	return config, nil
 }
 
+// Parse and validate a signal name or number
+func ParseSignal(rawSignal string) (syscall.Signal, error) {
+	// Strip off leading dash, to allow -1 or -HUP
+	basename := strings.TrimPrefix(rawSignal, "-")
+
+	signal, err := signal.ParseSignal(basename)
+	if err != nil {
+		return -1, err
+	}
+	if 1 <= signal && signal <= 64 { // FIXME: hardcoding SIGRTMAX
+		return signal, nil
+	}
+	return -1, fmt.Errorf("Invalid signal: %s", rawSignal)
```

Comment:
We generally prefer `errors.Errorf()` to `fmt.Errorf()` - they work identically, but the errors alias is used more consistently elsewhere

Reply:
Also, the linter is complaining that Invalid is capitalized, Go wants error messages to start with a lowercase letter